exercises/ansible_collections/README.md

```diff
@@ -0,0 +1,58 @@
+# Ansible Collections Workshop
+
+This workshop is a step by step series of exercises to demonstrate
+features and benefits of Ansible Collections, a unique distribution
+format of Ansible content that can be used to package and distribute
+playbooks, roles, modules, and plugins.
+
+## Time Planning
+
+The average duration of this workshop depends on the Ansible skills of
+practitioner. The average duration should be around 3 hours or less with
+advanced Ansible users.
+
+All exercises are self explanatory and guide the participants through the entire lab.
+Concepts are explained when they are introduced along with the practice exercise.
+
+## Prerequisites
+
+- Ansible v2.9
+
+- An OS of choice between RHEL/CentOS 8.x, CentOS Stream, Fedora 31+
+
+## Ansible Collections Exercises
+
+The workshop is structured in a set of progressive exercises, covering collections creation,
+usage from playbooks, roles and Tower and an introduction to Ansible Automation Hub.
+
+- [Exercise 1 - Create Collections](./1-create-collections)
+
+- [Exercise 2 - Collections from playbook](./2-collections-from-playbook)
+
+- [Exercise 3 - Collections from roles](./3-collections-from-roles)
+
+- [Exercise 4 - Collections from tower](./4-collections-from-tower)
+
+- [Exercise 7 - Use Automation Hub](./7-use-automation-hub)
```

Comment:
What't with the exercises 5 and 6? If they are not coming in a foreseeable future we should rename exercise 7, am I right?

Reply:
Yes, the plan was to have chapter 5 and 6 added immediately after the Hackfast - since that didn't happen until now, I'm changing the order.